rest/model/patch_test.go

```diff
@@ -224,3 +224,46 @@ func TestDownstreamTasks(t *testing.T) {
 	assert.Len(a.DownstreamTasks[0].Tasks, 2)
 	assert.Len(a.DownstreamTasks[0].VariantTasks, 1)
 }
+
+func TestPreselectedDisplayTasks(t *testing.T) {
```

Comment:
Would it be worth testing how it behaves where the user only selects a subset of execution tasks in the display task? I'm not sure if anyone really selects just a subset of a display task to run, but I don't think it's disallowed either, and may count as surprising/unexpected behavior.

Reply:
The execution tasks shouldn't be allowed to select on the UI (though I think the configure route could get behind that).

[This](https://github.com/evergreen-ci/evergreen/pull/8389/files#diff-66ceb3840b8d56d1344f5e203740f175ee47baaa942e8a9ca670b7b43bef619eR1389) test I think tests that though, it selects an execution tasks and makes sure the corresponding display task and other execution tasks are also paired (which propagates up to the variant tasks)